src/main/kotlin/com/github/schaka/janitorr/mediaserver/emby/EmbyProperties.kt

```diff
@@ -13,6 +13,7 @@ data class EmbyProperties(
         override val username: String,
         override val password: String,
         override val delete: Boolean = true,
+        override val excludeFavorited: Boolean = false,
```

Comment:
Are you sure this is going to compile?

The super class is missing the field that's supposed to be overridden here.